antlr4-wrapper.xml

```diff
@@ -32,14 +40,100 @@
     <property name="base-visitor-file" value="${target-package-dir}/${base-visitor-name}.java"/>
 
     <property name="listener-name" value="${lang-name}Listener"/>
-    <property name="listener-file" value="${target-package-dir}/${visitor-name}.java"/>
+    <property name="listener-file" value="${target-package-dir}/${listener-name}.java"/>
```

Comment:
I wonder, whether this fix has any implication... But we should see via japicmp, if we actually cleanup the listener for the first time as well and change it now. So far, the build is green, so we luckily are not impacted.

Reply:
The property was never used before, no incompatibilities from this. But there is more work for cleanup